index.html

```diff
@@ -0,0 +1,135 @@
+        </footer>
+      <section class="section-3">
+        <header>
+          <h2>Pricing</h2>
+        </header>
+          <div class="prices">
+            <h4>1: $.199</h4>
+            <h4>2-50: $1.49</h4>
+            <h4>51+: $0.99</h4>
+          </div>
+      </section>
+    </article>
+
+    <!-- SECOND PRODUCT SECTION -->
+    <article class="second-article">
+      <section class="section-1">
+        <header class="card-header">
+          <h1>Red Dunk Cards</h1>
+        </header>
+        <div>
+          <img class="red-img" src="images/red.png" alt="Red Card Image">
+        </div>
+        <div>
+          <p class="product-description">Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat. Ut wisi enim ad minim veniam, quis nostrud exerci tation.</p>
+        </div>
+        <div>
+          <p class="availability">Available</p>
+        </div>
+      </section>
+      <section class="section-2">
+        <header>
+          <h3>Specifications</h3>
+        </header>
+        <div class="specs">
+          <h4>Size: 4" x 4"</h4>
+          <h4>Weight: 8oz.<h4>
+        </div>
+        <footer class="product-footer">
+          <h6>Specifications valid until 04/01/2016</h6>
+        </footer>
+      <section class="section-3">
+        <header>
+          <h2>Pricing</h2>
+        </header>
+          <div class="prices">
+            <h4>1: $1.99</h4>
+            <h4>2-50: $1.49</h4>
+            <h4>51+: $0.99</h4>
+          </div>
+      </section>
+    </article>
+
+
+    <!-- THIRD PRODUCT SECTION -->
+    <article class="third-article">
+      <section class="section-1">
+        <header class="card-header">
+          <h1>Blue Dunk Cards</h1>
+        </header>
+        <div>
+          <img class="blue-img" src="images/blue.png" alt="Blue Card Image">
+        </div>
+        <div>
+          <p class="product-description">Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat. Ut wisi enim ad minim veniam, quis nostrud exerci tation.</p>
+        </div>
+        <div>
+          <p class="availability">Available</p>
+        </div>
+      </section>
+      <section class="section-2">
+        <header>
+          <h3>Specifications</h3>
+        </header>
+        <div class="specs">
+          <h4>Size: 4" x 4"</h4>
+          <h4>Weight: 8oz.<h4>
+        </div>
+        <footer class="product-footer">
+          <h6>Specifications valid until 04/01/2016</h6>
+        </footer>
+      <section class="section-3">
+        <header>
+          <h2>Pricing</h2>
+        </header>
+        <div class="prices">
+          <h4>1: $1.99</h4>
+          <h4>2-50: $1.49</h4>
+          <h4>51+: $0.99</h4>
+        </div>
+      </section>
+    </article>
+  <!-- END WRAPPER -->
+  </div>
+</body>
+</html>
```

Comment:
Like Github helpfully points out with its little 🚫  symbol, inserting a blank line at the end of a document is a convention from þe olde POXIX days.  Read more about it [here](http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline) if you'd like.